### ISSUE TITLE
Test Step should have default Est Time

### ISSUE THREAD

**Issue by adeelrehman007** [AutomationSolutionz/Framework_0.1#595]
Default value should be 30 seconds.

Issue only when creating test step on fly

**Comment by m3thun** [comment]
@riz-hossain labels pls!

**Comment by riz-hossain** [comment]
This is done Put it under Minar name and close it.                                                                                                                                                                                                                                                                                                                                          Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: mithunSent: Wednesday, August 19, 2015 12:28 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Test Step should have default Est Time (#595)@riz-hossain labels pls!

—Reply to this email directly or view it on GitHub.

**Comment by riz-hossain** [comment]
Actually I thought this was really old.  Default value can be changed from going into test step itself.                                                                                                                                                                                                                                                                                                                                         Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: Riz HossainSent: Wednesday, August 19, 2015 12:29 AMTo: mithun; AutomationSolutionz/Framework_0.1Subject: Re: [Framework_0.1] Test Step should have default Est Time (#595)                                                                                      This is done Put it under Minar name and close it.                                                                                                                                                                                                                                                                                                                                          Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: mithunSent: Wednesday, August 19, 2015 12:28 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Test Step should have default Est Time (#595)@riz-hossain labels pls!

—Reply to this email directly or view it on GitHub.

**Comment by adeelrehman007** [comment]
This shouldn't be closed. The issue isn't the default value of the step.
It's the fact that when the test step is written on the fly, the est time
field isn't populated automatically as it is when a pre-defined test step
is added.

Please re-open ticket

**Comment by m3thun** [comment]
@riz-hossain Please reply to the comment made by Adeel.

**Comment by m3thun** [comment]
For now, I re-opened it and removing the label "Done".

**Comment by riz-hossain** [comment]
To clarify, As soon as you hit + to add a new step, before you start typing any names for any steps, user will be prefilled with some time.   We can fill up the descriptions and expected as well with N/A... this way it will help manual testers just write the new steps and nothing else. This was a request that came from few others. This will be a low priority and fr and will be assigned to Minar.  It will be targeted for release version 1.0 as it increases user experience for                                                                                                                                                                                                                                                                                                                                         Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: mithunSent: Wednesday, August 19, 2015 8:56 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Test Step should have default Est Time (#595)For now, I re-opened it and removing the label "Done".

—Reply to this email directly or view it on GitHub.

**Comment by adeelrehman007** [comment]
Correct. Either the field can be prefilled 'before' the user types any step
name OR once it realizes that the test step is being created on the fly
(i.e. is not a already a saved step)
